irksome/stepper.py

```diff
@@ -76,6 +84,28 @@ def TimeStepper(F, butcher_tableau, t, dt, u0, **kwargs):
             F, butcher_tableau, t, dt, u0, bcs, appctx=appctx,
             solver_parameters=solver_parameters, nullspace=nullspace,
             bc_type=bc_type, splitting=splitting)
+    elif stage_type == "adapt":
+        bcs = kwargs.get("bcs")
+        bc_type = kwargs.get("bc_type", "DAE")
+        splitting = kwargs.get("splitting", AI)
```

Comment:
Do we need to consider allowing other types of BC, splittings, etc?  I get that this is a first stab.  It seems like maybe adaptivity is something that could be built on top of any of those approaches eventually.

Reply:
This is just copied from the `deriv` case.  Just setting defaults here, not limiting selections, I think.